### DISCUSSION TITLE
Templates for draft issues

### DISCUSSION THREAD

**Discussion by luccahora** [community/community#50026] [New to GitHub]
I'm using Github Projects to track tasks. I wanted to know if it is possible to create a Templates for draft issues?

**Comment by Anmol-Baranwal** [comment]
Yes, it is possible to create templates for draft issues in GitHub Projects. You can create a template for draft issues by creating a markdown file with the necessary fields and instructions.

Here are the steps to create a template for draft issues:

1. Create a new file in your repository with the `.md` extension. For example, you could name the file `draft_issue_template.md`.

2. Add the necessary fields and instructions for your draft issue. You can use markdown to format the text and add headings, lists, and other formatting. Here is an example template:

```markdown
# Draft Issue Template

**Description:** 

Write a brief description of the issue here.

**Steps to Reproduce:**

1. 
2. 
3. 

**Expected Behavior:**

Write a description of what should happen.

**Actual Behavior:**

Write a description of what is happening.

**Additional Information:**

Add any additional information or context here.
```

3. Save the file to your repository.

4. When creating a new issue, select the "Draft" issue template from the dropdown list.

5. The fields and instructions from your template will be pre-populated in the new issue.

By creating a template for draft issues, you can ensure that all issues follow a consistent format and include all the necessary information. This can help streamline your project management process and make it easier for team members to create and track issues.

**Reply by Anmol-Baranwal** [reply]
@luccahora 
I just wanted to check in and see if you had any further questions or concerns about my answer on Github. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by Narigo** [reply]
This answer doesn't work for me, if I understood the initial request correctly.

When I'm in GitHub projects and create a new item, the description field is always empty. When using "convert to issue" there is no option to select a template and whatever text I've put into the draft will be in the new issue. It basically circumvents the issue templates and this makes creating new issues via GitHub projects very cumbersome.

**Reply by YaroShkvorets** [reply]
> When I'm in GitHub projects and create a new item, the description field is always empty. When using "convert to issue" there is no option to select a template and whatever text I've put into the draft will be in the new issue. It basically circumvents the issue templates and this makes creating new issues via GitHub projects very cumbersome.

Have you found a way to solve this?

**Reply by joaomas** [reply]
> This answer doesn't work for me, if I understood the initial request correctly.
> 
> When I'm in GitHub projects and create a new item, the description field is always empty. When using "convert to issue" there is no option to select a template and whatever text I've put into the draft will be in the new issue. It basically circumvents the issue templates and this makes creating new issues via GitHub projects very cumbersome.

I have the same problem it's your same need

**Comment by YaroShkvorets** [comment]
Duplicates: 
https://github.com/orgs/community/discussions/46167
https://github.com/orgs/community/discussions/14010
https://github.com/orgs/community/discussions/6481

**Comment by anahelenasilva** [comment]
I guess what @luccahora meant is that we have a template for a draft task in a board, not an issue, like this:
![image](https://github.com/community/community/assets/10438122/0f8512ec-98d5-416e-8181-485ea50eb189)

**Comment by anahelenasilva** [comment]
@Anmol-Baranwal do you know if it's possible to do what we're trying to do, which is create a template to use on draft tasks in a board?

**Reply by Anmol-Baranwal** [reply]
I have use GitHub projects I was making my project, but I'm not really sure as of current date. Maybe you can search it in the docs, it would definitely be there. GitHub Team likes to do lot of modifications :)

**Reply by anahelenasilva** [reply]
I did search it in the docs, but couldn't find it :/ maybe it's not available at all. Anyways, thanks for your response! :)

**Reply by Anmol-Baranwal** [reply]
Sure, no worries. Thanks for letting me know.

**Comment by zbrown001** [comment]
Is there any update on this? Our team would really love this feature
